Jenkinsfile

```diff
@@ -0,0 +1,114 @@
+// This Jenkinsfile is used by Jenkins to run the DiagramConverter step of Reactome's release.
+// It requires that the 'GenerateGraphDatabaseAndAnalysisCore' step has been run successfully before it can be run.
+// This is the FINAL step that makes updates to any database.
+
+import org.reactome.release.jenkins.utilities.Utilities
+
+// Shared library maintained at 'release-jenkins-utils' repository.
+def utils = new Utilities()
+
+pipeline{
+	agent any
+
+	// Set output folder name where files generated by step will be output.
+    	environment {
+        	OUTPUT_FOLDER = "diagram"
+    	}
+	stages{
+		// This stage checks that upstream project 'GenerateGraphDatabaseAndAnalysisCore' was run successfully.
+		stage('Check Graph DB & Analysis Core build succeeded'){
+			steps{
+				script{
+                    			utils.checkUpstreamBuildsSucceeded("GenerateGraphDatabaseAndAnalysisCore")
+				}
+			}
+		}
+		// This stage builds the jar file using maven.
+		stage('Setup: Build jar file'){
+			steps{
+				script{
+					sh "mvn clean package"
+				}
+			}
+		}
+		// Execute the jar file, producing the diagram JSON files.
+		stage('Main: Run Diagram-Converter'){
+			steps{
+				script{
+				    def releaseVersion = utils.getReleaseVersion()
+					sh "mkdir -p ${env.OUTPUT_FOLDER}"
+					withCredentials([usernamePassword(credentialsId: 'mySQLUsernamePassword', passwordVariable: 'mysqlPass', usernameVariable: 'mysqlUser')]){
+						withCredentials([usernamePassword(credentialsId: 'neo4jUsernamePassword', passwordVariable: 'neo4jPass', usernameVariable: 'neo4jUser')]){
+				 			sh "java -jar target/diagram-converter-jar-with-dependencies.jar --graph_user $neo4jUser --graph_password $neo4jPass --rel_user $mysqlUser --rel_password $mysqlPass --rel_database ${env.REACTOME_DB} --output ./${env.OUTPUT_FOLDER}"
+					        	// Create archive that will be stored on S3.
+							sh "tar -zcf diagrams-v${releaseVersion}.tgz ${env.OUTPUT_FOLDER}/"
+							// Restart tomcat7 and neo4j services after updates were made to graph db.
+					        	sh "sudo service tomcat7 stop"
+							sh "sudo service neo4j stop"
+							sh "sudo service neo4j start"
+							sh "sudo service tomcat7 start"
```

Comment:
I think you can do these as "restart" rather than "stop" and "start"?

Reply:
Yea I think I will add that to the `visudo` commands allowed for Jenkins. Originally I thought that keeping it as discrete start/stops would be better, but there isn't much of a reason for that